scripts/src/test-runner.js

```diff
@@ -44,6 +44,40 @@ let userDetails = JSON.parse(file);
 // -- Create Thunder object --
 let thunder = new ThunderClient(args.endpoint, auth.application, auth.secret);
 
+// -- Define response handler --
+/**
+ * Handles a response from Thunder and prints out necessary information.
+ *
+ * @param {Error} err - The error object that was returned from the Thunder call.
+ * @param {object} result - The result that was returned from the Thunder call.
+ * @param {string} methodName - The name of the method that was called.
+ * @param {function} callback - The function to call after validation and logging.
+ * @return When the validation is complete.
+ */
+function handleResponse(err, result, methodName, callback) {
+  if (err) {
+    console.log('An error occurred while performing method %s', methodName);
+
+    if (args.verbose) {
+      console.log('Details:');
+      console.log(result);
+    }
+
+    console.log('\n');
+    return callback(err);
+  }
+
+  console.log('Successfully completed method %s', methodName);
+
+  if (args.verbose) {
+    console.log('Response:');
+    console.log(result);
+  }
+
+  console.log('\n');
+  callback(null, result);
```

Comment:
Are there function types? How do you know this take two params?

Reply:
Unfortunately, JS is very loosely typed so there is no function type. The best you can get is through documentation (JSDoc comments). I'll improve the comments more as these files are iterated on.